### PR TITLE
DM-39576: buttons to pause and resume mirror raising and lowering

### PR DESCRIPTION
**Pull request by pkubanek**
- Moved Panic button
- -> None (mypy fix)
- Pause/Resume raising/lowering buttons

### CODE REVIEW

File: python/lsst/ts/criopy/m1m3/ApplicationControlWidget.py [hunk 1]
```diff
@@ -250,19 +255,35 @@ def __init__(self, m1m3: MetaSAL):
...
                 self.TEXT_ENABLE,
                 self.TEXT_RAISE,
+                self.TEXT_PAUSE_RAISING,
                 self.TEXT_ENTER_ENGINEERING,
                 self.TEXT_STANDBY,
             ],
             {
                 self.TEXT_RAISE: "raiseM1M3",
+                self.TEXT_PAUSE_RAISING: "pauseM1M3RaisingLowering",
+                self.TEXT_RESUME_RAISING: "resumeM1M3RaisingLowering",
```
Comment (teweitsai) [inline python/lsst/ts/criopy/m1m3/ApplicationControlWidget.py:265]: "pauseM1M3RaisingLowering" -> "pauseM1M3Raising" ?
"resumeM1M3RaisingLowering" -> "resumeM1M3Raising" ?

File: python/lsst/ts/criopy/m1m3/ApplicationControlWidget.py [hunk 2]
```diff
@@ -250,19 +255,35 @@ def __init__(self, m1m3: MetaSAL):
...
                 self.TEXT_ABORT_RAISE: "abortRaiseM1M3",
                 self.TEXT_LOWER: "lowerM1M3",
+                self.TEXT_PAUSE_LOWERING: "pauseM1M3RaisingLowering",
+                self.TEXT_RESUME_LOWERING: "resumeM1M3RaisingLowering",
```
Comment (teweitsai) [inline python/lsst/ts/criopy/m1m3/ApplicationControlWidget.py:269]: "pauseM1M3RaisingLowering" -> "pauseM1M3Lowering"
"resumeM1M3RaisingLowering" -> "resumeM1M3Lowering"

**Review by teweitsai — APPROVED** [review]
Look good!

File: python/lsst/ts/criopy/m1m3/ApplicationControlWidget.py (hunk 1 above)
Comment (pkubanek) [reply python/lsst/ts/criopy/m1m3/ApplicationControlWidget.py:265]: This is SAL/DDS command, so that cannot be changed.

**Pull request merged**